client/my-sites/theme/theme-related-themes-card/index.jsx

```diff
@@ -0,0 +1,86 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import i18n from 'lib/mixins/i18n';
+import SectionHeader from 'components/section-header';
+import { getDetailsUrl } from 'my-sites/themes/helpers';
+
+var ThemeRelatedThemesCard = React.createClass( {
+
+	propTypes: {
+		currentTheme: React.PropTypes.string.isRequired
+	},
+
+	getRelatedThemes() {
+		var themes = [
+			{ name: 'Twenty Sixteen', slug: 'twentysixteen' },
+			{ name: 'Rowling', slug: 'rowling' },
+			{ name: 'Hemingway Rewritten', slug: 'hemingway-rewritten' },
+			{ name: 'Gazette', slug: 'gazette' },
+			{ name: 'Ecto', slug: 'ecto' },
+			{ name: 'Isola', slug: 'isola' },
+			{ name: 'Edin', slug: 'edin' },
+			{ name: 'Sela', slug: 'sela' },
+			{ name: 'Pique', slug: 'pique' },
+			{ name: 'Harmonic', slug: 'harmonic' },
+		];
+
+		var selectedThemes = [];
```

Comment:
Use `let` here. We try to avoid `var` (which is ES5, and less specific) these days and use `const` or `let` instead.